### ISSUE TITLE
sap_install_media_detect: Add an option for skipping handling RAR files

### ISSUE THREAD

**Issue by berndfinger** [sap-linuxlab/community.sap_install#855]
We observed that when running the role on a RHEL system with the file `/etc/yum/vars/releasever` containing the string `9.4`, the installation of the `unar` package from EPEL fails:

```
TASK [community.sap_install.sap_install_media_detect : SAP Install Media Detect - Prepare - EPEL - Install the unar package] ***************
fatal: [hana-p]: FAILED! => changed=false 
  msg: 'Failed to download metadata for repo ''epel'': Cannot prepare internal mirrorlist: Status code: 404 for https://mirrors.fedoraproject.org/metalink?repo=epel-9.4&arch=x86_64&infra=$infra&content=$contentdir (IP: 1.2.3.4)'
  rc: 1
  results: []
```

This issue can be worked around by installing a package for handling rar files separately and setting the role parameter `sap_install_media_detect_rar_package` accordingly.

As the `unar` package is only required for listing the contents of, and extracting, self-extracting RAR files, we should however implement an option for ignoring the RAR handling.

**Comment by berndfinger** [comment]
BTW We already have a tag for the tasks which are handling the RAR files: `sap_install_media_detect_rar_handling`

**Comment by ja9fuchs** [comment]
Just to be clear, the main issue seems to be either a bug in the epel repo rpm that ships files with `$releasever` in the mirror urls, or it is an issue with the mirror routes not providing the path for RHEL 9.4 specifically. 
It partially works (for the one epel repo we use, out of several in the config) on e.g. 9.2.

**Comment by berndfinger** [comment]
Solved by #856.
